docs/source/guides/user/chapters/introduction.rst

```diff
@@ -304,6 +304,9 @@ output in the standard output of the runner, simply use::
           one half starting from the beginning of the content, the other
           half from the end of the content.
 
+.. note:: The avocado xunit format adds some attributes which are not a part of 
+          the official format, but they are used by `GitLabCI <https://gitlab.com/gitlab-org/gitlab/-/blob/master/lib/gitlab/ci/parsers/test/junit.rb#L83>`__.
```

Comment:
This reference can rot very soon.  Please update it with a permanent reference, that is, instead of using the branch name, use a specific commit instead.